### ISSUE TITLE
The C compiler identification is unknown

### ISSUE THREAD

**Issue by hynguyen2610** [dotnet/coreclr#2524]
I have VS 2015 Community with C++ module installed. Cmake installed and added to PATH. When run the build.bat from an empty new cmd windows, I got this error:
Checking pre-requisites...

Commencing build of native components for Windows_NT.x64.Debug

-- The C compiler identification is unknown
-- The CXX compiler identification is unknown
CMake Error at CMakeLists.txt:5 (project):
  No CMAKE_C_COMPILER could be found.

**Comment by mikedn** [comment]
Maybe you installed an older version of CMake which doesn't know about VS2015? Works fine here with CMake 3.3.0.

**Comment by hynguyen2610** [comment]
I am using the version 3.4.1

**Comment by gkhanna79** [comment]
@janvorli Is this similar to the issue you are fixing?

**Comment by janvorli** [comment]
@gkhanna79 Doesn't seem to be related.

**Comment by janvorli** [comment]
Can you please try to create and build a simple C/C++ project in the VS2015 to see if it works? I have seen people reporting that they sometimes needed to create one to get the C++ compiler installed.

**Comment by hynguyen2610** [comment]
Hi,
I can build C++ project normally. In the documentation said you are using CMake 3.0.2, when I using this version, it's not recognize my VS 2015, I tried version 3.4.1, the error is  "The C compiler identification is unknown" like the post, I try to using Cmake 3.3.2, its don't fail into compiler problem anymore. The build can run a long way.
I am running the build.bat with VS Developer x64 Command Promt now, sometime I got an error "Cannot connect to child process" when the building is continueing, may be I will post this in another issue
